pkg/errors/stacktrace.go

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) The Thanos Authors.
+// Licensed under the Apache License 2.0.
+
+package errors
+
+import (
+	"fmt"
+	"runtime"
+	"strings"
+)
+
+// stack holds the address of
```

Comment:
I think this would be ` holds a snapshot of program counters.`

Reply:
Yep. Missed